Jenkinsfile

```diff
@@ -1,150 +1,140 @@
 max_time = 180
 
+setup_pip_venv = """
+    rm -rf venv
+    conda list
+
+    python3 -m venv venv
+    source venv/bin/activate
+    python3 -m pip install -U pip setuptools wheel
+
+    python3 -m pip install 'graphviz'
+    python3 -m pip install 'jupyter-sphinx>=0.2.2'
+    python3 -m pip install 'portalocker'
+    python3 -m pip install 'scipy>=1.3.3,<1.5.0'
+    python3 -m pip install 'nose'
+    python3 -m pip install 'docutils'
+    python3 -m pip install 'mu-notedown'
+    python3 -m pip install 'flake8'
+    python3 -m pip install 'awscli>=1.18.140'
+
+    export MPLBACKEND=Agg
+"""
+
+setup_mxnet_gpu = """
+    python3 -m pip install mxnet-cu101==1.7.0
+    export LD_LIBRARY_PATH=/usr/local/cuda-10.1/lib64
+    export MXNET_CUDNN_AUTOTUNE_DEFAULT=0
+"""
+
+cleanup_venv = """
+    deactivate
+    rm -rf venv
+"""
+
 stage("Unit Test") {
   parallel 'core': {
     node('linux-cpu') {
-      ws('workspace/autogluon-core-py3') {
+      ws('workspace/autogluon-core-py3-v3') {
         timeout(time: max_time, unit: 'MINUTES') {
           checkout scm
           VISIBLE_GPU=env.EXECUTOR_NUMBER.toInteger() % 8
           sh """#!/bin/bash
           set -ex
-          conda env update -n autogluon_core_py3 -f docs/build.yml
-          conda activate autogluon_core_py3
+          conda env update -n autogluon-core-py3-v3 -f docs/build.yml
+          conda activate autogluon-core-py3-v3
           conda list
-          export CUDA_VISIBLE_DEVICES=${VISIBLE_GPU}
+          ${setup_pip_venv}
+          python3 -m pip install 'mxnet==1.7.0.*'
```

Comment:
Why are we using CPU mxnet?

Reply:
because this is `linux-cpu` node (2 tasks run on it)